src/mess/drivers/apple2.c

```diff
@@ -1212,7 +1212,7 @@ static SLOT_INTERFACE_START(apple2_cards)
 	SLOT_INTERFACE("ap16alt", A2BUS_IBSAP16ALT)    /* IBS AP16 (German VideoTerm clone), alternate revision */
 	SLOT_INTERFACE("vtc1", A2BUS_VTC1)    /* Unknown VideoTerm clone #1 */
 	SLOT_INTERFACE("vtc2", A2BUS_VTC2)    /* Unknown VideoTerm clone #2 */
-	SLOT_INTERFACE("arcbd", A2BUS_ARCADEBOARD)    /* Third Millenium Engineering Arcade Board */
```

Comment:
Another one of theses - you're wasting your own time as well as others by 'correcting' the spelling of a trademark.